examples/qt_types_standalone/src/main.cpp

```diff
@@ -241,6 +241,41 @@ TEST_CASE("Can copy a value QPointF on the Rust side")
   CHECK(c.y() == 4.56);
 }
 
+TEST_CASE("Can construct a QRectF on the Rust side")
+{
+  const auto r = construct_qrectf();
+  CHECK(r.x() == 1.23);
```

Comment:
Maybe we should be using epsilon-based checks here as well? Like we do in read_qrectf().
Theoretically these floating point numbers can compile down to slightly different binary representations, so they won't be exactly equal :thinking:

Reply:
Fixed to use `qFuzzyCompare` and then added a commit to change the QPointF and QSizeF instances.

Reply:
:+1: